app/utils/versionChecker.js

```diff
@@ -1,18 +1,15 @@
 class VersionChecker {
-  latest (callback) {
```

Comment:
There was a problem with sinon, it wasn't restored after tests. Usage of sandbox fixed the problem.